ui/app/templates/components/identity/edit-form.hbs

```diff
@@ -24,7 +24,36 @@
       />
     {{/if}}
     {{#each this.model.fields as |attr|}}
-      <FormField data-test-field={{true}} @attr={{attr}} @model={{@model}} />
+      <FormField data-test-field={{true}} @attr={{attr}} @model={{@model}}>
+        <div class="form-section">
+          {{#if this.model.canCreatePolicies}}
+            <SearchSelectWithModal
+              @id="policies"
+              @label="Policies"
+              @labelClass="title is-4"
+              @models={{array "policy/acl" "policy/rgp"}}
+              @inputValue={{@model.policies}}
+              @onChange={{action (mut this.model.policies)}}
+              @onCreate={{action "createSearchSelectModel"}}
+              @formModel={{this.policyModel}}
+              @fallbackComponent="string-list"
+              @modalFormComponent="policy-form"
+              @excludeOptions={{array "root"}}
```

Comment:
added this since `root` can't be added as a policy

Reply:
The `SearchSelect` doesn't accept `excludeOptions` as an arg - I could add it...but the scope of this PR is already 😬 

Since this was how it originally functioned I felt like it was ok to leave 😓